modules/dates.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from './luxon.js';
+
+export default function displayCurrentDate() {
+  const currentDate = DateTime.now().toLocaleString(DateTime.DATETIME_SHORT);
+  const dateContainer = document.getElementById('date');
+  dateContainer.innerHTML = `<small>${currentDate}</small>`;
+}
```

Comment:
- [OPTIONAL] Great job implementing the required changes from the previous reviewer. It is not bad too if this function could follow the same way. ES6 syntax rather than function keyword